src/content/docs/en/reference/api-reference.mdx

```diff
@@ -537,6 +537,20 @@ import Shout from "../components/Shout.astro";
 <!-- renders as <div>SLOTS!</div> -->
 ```
 
+Callback functions can be passed to named slots inside a wrapping HTML element tag with a `slot` attribute. This element is only used to transfer the callback to a named slot and will not be rendered onto the page.
+
+A callback 
+
+```astro
+<Shout message="slots!">
+  <fragment slot="message">
+    {(message) => <div>{message}</div>}
+  </fragment>
+</Shout>
+```
+
+This tag cannot be named `Fragment`, `slot` or include a dash `-`. The tag is only used to transfer the callback to a named slot, it does not get rendered onto the page.
```

Comment:
```suggestion
Use a standard HTML element for the wrapping tag, or any lower case tag (e.g. `<fragment>` instead of `<Fragment />`) that will not be interpreted as a component.  Do not use the HTML `<slot>` element as this will be interpreted as an Astro slot.
```
I realized that I don't think `<fragment>` *is* a standard HTML element :sweat_smile: 

So, we should *either* do something like this, OR we should change the code sample to use a standard element!

Reply:
That example LGTM

I typically use a non-standard element like `<fragment>` for these examples to avoid confusion about the tag not getting rendered onto the page and looking very similar to other patterns inside `.astro` files like a `.map()`